ci/compile_py_proto/install_py_proto.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+
+mkdir -p "${BUILD_WORKSPACE_DIRECTORY}"/python/ray/core/generated
+mkdir -p "${BUILD_WORKSPACE_DIRECTORY}"/python/ray/serve/generated
+
+tar -xvf "$1" -C "${BUILD_WORKSPACE_DIRECTORY}"/python/ray/core/generated
+mv "${BUILD_WORKSPACE_DIRECTORY}"/python/ray/core/generated/serve_*.py "${BUILD_WORKSPACE_DIRECTORY}"/python/ray/serve/generated
```

Comment:
this move feels a bit weird honestly, maybe save them in two different tarballs?